runtime/gc_modron_startup/mminit.cpp

```diff
@@ -218,16 +218,26 @@ initializeMutatorModelJava(J9VMThread* vmThread)
 void
 cleanupMutatorModelJava(J9VMThread* vmThread)
 {
-	J9VMDllLoadInfo* loadInfo;
-	J9JavaVM* vm = vmThread->javaVM;
 	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread);
 
 	if (NULL != env) {
+		J9VMDllLoadInfo *loadInfo = NULL;
+		J9JavaVM *vm = vmThread->javaVM;
+
+#if defined(OMR_MIXED_REFERENCES_MODE_STATIC)
+		if (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm)) {
+			loadInfo = FIND_DLL_TABLE_ENTRY(J9_GC_DLL_NAME);
+		} else {
+			loadInfo = FIND_DLL_TABLE_ENTRY(J9_GC_FULL_DLL_NAME);
+		}
+#else
+		loadInfo = FIND_DLL_TABLE_ENTRY(J9_GC_DLL_NAME);
+#endif /* defined(OMR_MIXED_REFERENCES_MODE_STATIC) */
+
```

Comment:
This pattern repeats multiple places. Does it make sense to create function for it?

Reply:
Added a helper in this file